liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -0,0 +1,789 @@
+            }
+
+            cliLogLevel = Level.OFF;
+        }
+
+        rootLogger.setLevel(logLevel);
+        liquibaseLogger.setLevel(logLevel);
+
+        for (Handler handler : rootLogger.getHandlers()) {
+            if (handler instanceof ConsoleHandler) {
+                handler.setLevel(cliLogLevel);
+            }
+
+            handler.setFilter(new SecureLogFilter(logService.getFilter()));
+        }
+    }
+
+    private CommandLine getRootCommand(CommandLine commandLine) {
+        while (commandLine.getParent() != null) {
+            commandLine = commandLine.getParent();
+        }
+        return commandLine;
+    }
+
+    private Map<String, Object> configureResourceAccessor(ClassLoader classLoader) {
+        Map<String, Object> returnMap = new HashMap<>();
+
+        returnMap.put(Scope.Attr.resourceAccessor.name(), new CompositeResourceAccessor(new FileSystemResourceAccessor(Paths.get(".").toAbsolutePath().toFile()), new CommandLineResourceAccessor(classLoader)));
+
+        return returnMap;
+    }
+
+    protected ClassLoader configureClassLoader() throws IllegalArgumentException {
+        final String classpath = IntegrationConfiguration.CLASSPATH.getCurrentValue();
+
+        final List<URL> urls = new ArrayList<>();
+        if (classpath != null) {
+            String[] classpathSoFar;
+            if (isWindows()) {
+                classpathSoFar = classpath.split(";");
+            } else {
+                classpathSoFar = classpath.split(":");
+            }
+
+            for (String classpathEntry : classpathSoFar) {
+                File classPathFile = new File(classpathEntry);
+                if (!classPathFile.exists()) {
+                    throw new IllegalArgumentException(classPathFile.getAbsolutePath() + " does.not.exist");
+                }
+
+                try {
+                    URL newUrl = new File(classpathEntry).toURI().toURL();
+                    Scope.getCurrentScope().getLog(getClass()).fine(newUrl.toExternalForm() + " added to class loader");
+                    urls.add(newUrl);
+                } catch (MalformedURLException e) {
+                    throw new IllegalArgumentException(e);
+                }
+            }
+        }
+
+        final ClassLoader classLoader;
+        if (IntegrationConfiguration.INCLUDE_SYSTEM_CLASSPATH.getCurrentValue()) {
+            classLoader = AccessController.doPrivileged((PrivilegedAction<URLClassLoader>) () -> new URLClassLoader(urls.toArray(new URL[0]), Thread.currentThread()
+                    .getContextClassLoader()));
+
+        } else {
+            classLoader = AccessController.doPrivileged((PrivilegedAction<URLClassLoader>) () -> new URLClassLoader(urls.toArray(new URL[0]), null));
+        }
+
+        Thread.currentThread().setContextClassLoader(classLoader);
+
+        return classLoader;
+    }
+
+    private void addSubcommand(CommandDefinition commandDefinition, CommandLine commandLine) {
```

Comment:
camelCasing subCommand

Reply:
we're counting "subcommand" as one word, not two so shouldn't be camel cased